src/TRoomDB.cpp

```diff
@@ -252,21 +258,51 @@ TArea * TRoomDB::getArea( int id )
     }
 }
 
-void TRoomDB::setAreaName( int areaID, QString name )
+bool TRoomDB::setAreaName( int areaID, QString name )
 {
+    if( areaID < 1 ) {
+        qWarning( "TRoomDB::setAreaName((int)areaID, (QString)name): WARNING: Suspect areaID: %n supplied.", areaID );
```

Comment:
I felt it was advisable because there are multiple setAreaName(...) and announcing the signature would help to determine where the qWarning was coming from.  Given that the variable arguments supplied to the qWarning come from those parameters I think we can assume that anyone changing the signature would spot the desirability of changing the text - or perhaps not given the state of some of the TLuaInterpreter diagnostic message for the user. :see_no_evil: :hear_no_evil: :speak_no_evil: